salt/metalk8s/defaults.yaml

```diff
@@ -1,3 +1,29 @@
+kubeadm_preflight:
+  mandatory:
+    packages:
+      - util-linux            # provides nsenter, mount
+      - iproute               # provides ip
+      - iptables              # provides iptables
+    ports:
+      - 10250
+      - 10251
+      - 10252
+      - 2379
+      - 2380
+    services:
+      running:
+        - containerd
```

Comment:
We should not use this key like for the container-runtime-endpoint you should use the key `kubelet.container_engine` which is also not in this PR :)